app/views/unauthorisedEori.scala.html

```diff
@@ -0,0 +1,44 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.ExternalServicesConfig
+@import views.helpers.Title
+@import views.html.components.gds._
+
+@this(
+    govukLayout: gdsMainTemplate,
+    pageTitle: pageTitle,
+    paragraphBody: paragraphBody,
+    link: link,
+    bulletList: bulletList,
+    config: ExternalServicesConfig
+)
+
+@(displaySignOut: Boolean)(implicit request: Request[_], messages: Messages)
```

Comment:
Can remove `displaySignOut` parameter for this. Unauthorised EORI means the user will always be logged in so we can always show them the sign out link on this page.